README.adoc

```diff
@@ -414,68 +416,64 @@ endif::[]
-It appears in a format similar to the following:
-
-```
-Node:         10.114.85.140/10.114.85.140
-Node-Selectors:  <none>
+Open another command-line session by selecting **Terminal** > **New Terminal** from the menu of the IDE.
+Run the following commands to store the proxy path of the **system** and **inventory** services.
 ```
-
-The IP for the system service is `10.114.85.140`.
-Store the IP in a variable.
-
-```
-SYSTEM_HOST={system-node-ip}
+NAMESPACE_NAME=`bx cr namespace-list | grep sn-labs- | sed 's/ //g'`
+SYSTEM_PROXY=localhost:8001/api/v1/namespaces/$NAMESPACE_NAME/services/system-service/proxy
+INVENTORY_PROXY=localhost:8001/api/v1/namespaces/$NAMESPACE_NAME/services/inventory-service/proxy
 ```
 {: codeblock}
 
-Use another `kubectl` command to get the node IP for the `inventory` service:
+Run the following echo commands to verify the variables:
 
 ```
-kubectl describe pod inventory | grep Node
+echo $SYSTEM_PROXY && echo $INVENTORY_PROXY
 ```
 {: codeblock}
 
-Store this IP in a variable as well.
+
+The output appears similar to the following:
 
 ```
-INVENTORY_HOST={inventory-node-ip}
+localhost:8001/api/v1/namespaces/sn-labs-yourname/services/system-service/proxy
+localhost:8001/api/v1/namespaces/sn-labs-yourname/services/inventory-service/proxy
 ```
-{: codeblock}
 
-Verify that the variables that contain the IPs are set correctly:
+Then use the following **curl** command to access your **system** microservice:
 
 ```
-echo $SYSTEM_HOST && echo $INVENTORY_HOST
+curl http://$SYSTEM_PROXY/system/properties | jq
 ```
 {: codeblock}
 
-Then use the following `curl` commands to access your microservices:
+and use the following **curl** command to access your **inventory** microservice:
 
 ```
-curl http://$SYSTEM_HOST:31000/system/properties
+curl http://$INVENTORY_PROXY/inventory/systems/system-service | jq
 ```
 {: codeblock}
 
+The first URL returns system properties and the name of the pod in an HTTP header called **X-Pod-Name**.
+To view the header, you can use the **-I** option in the **curl** command when you make a request to the
+**http://$SYSTEM_PROXY/system/properties** URL.
+
 ```
-curl http://$INVENTORY_HOST:32000/inventory/systems/system-service
+curl -I http://$SYSTEM_PROXY/system/properties
 ```
 {: codeblock}
 
-The first URL returns system properties and the name of the pod in an HTTP header called `X-Pod-Name`.
-To view the header, you can use the `-I` option in the `curl` command when you make a request to the
-`http://$SYSTEM_HOST:31000/system/properties` URL.
-The second URL adds properties from `system-service` to the inventory {kube} Service. 
-Making a request to the `http://$INVENTORY_HOST:32000/inventory/systems/[kube-service]` URL in general 
-adds to the inventory depending on whether `kube-service` is a valid {kube} service that can be accessed.
+The second URL adds properties from **system-service** to the inventory {kube} Service. 
```

Comment:
Change "The second URL" to "The `http://$INVENTORY_PROXY/inventory/systems/system-service` URL to avoid confusion regarding what URL is being referenced. Also, add a noun after **system-service" to indicate what is being referenced, so change "from **system-service** to something like "from the **system-service** directory".